scripts/common/common.sh

```diff
@@ -286,6 +286,13 @@ function kubernetes_resource_exists() {
     kubectl -n "$namespace" get "$kind" "$name" &>/dev/null
 }
 
+function kubernetes_cluster_resource_exists() {
+    local kind=$1
+    local name=$2
+
+    kubectl -n "$namespace" get "$kind" "$name" &>/dev/null
+}
+
```

Comment:
Isn't this the same thing as the above `kubernetes_resource_exists`? Maybe you wanted to remove the NS field?